### PR TITLE
Fixed subprocess in setup.py. It was causing setup to fail.

### PR DESCRIPTION
**Pull request by mingjie-intel**
To prevent timeouts of hefty subprocess calls, redirect subprocess's output to DEVNULL.

- [x] Have you provided a meaningful PR description?
- [x] Have you added a test, reproducer or referred to an issue with a reproducer?
N/A
- [x] Have you tested your changes locally for CPU and GPU devices?
- [x] Have you made sure that new changes do not introduce compiler warnings?
- [x] If this PR is a work in progress, are you filing the PR as a draft?

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -130,9 +130,10 @@ def spirv_compile():
         "numba_dpex/ocl/atomics/atomic_ops.spir",
         "numba_dpex/ocl/atomics/atomic_ops.bc",
     ]
-    subprocess.check_call(clang_args, stderr=subprocess.STDOUT, shell=False)
-    subprocess.check_call(spirv_args, stderr=subprocess.STDOUT, shell=False)
-
+    print(clang_args)
```
Comment (oleksandr-pavlyk) [inline setup.py:133]:
```suggestion
```

File: setup.py [hunk 2]
```diff
@@ -130,9 +130,10 @@ def spirv_compile():
...
+    print(spirv_args)
```
Comment (oleksandr-pavlyk) [inline setup.py:134]:
```suggestion
```

**Review by oleksandr-pavlyk — APPROVED** [review]
If we are not using neither stdout nor stderr, it is the right thing to do.

**Comment by diptorupd** [comment]
I have not been able to replicate the issue for a while. I am closing the PR for now. We can revisit if the issue crops back.

**Pull request merged**